command/tests/sample_program_websocket_test/test_websocket_program.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	"os"
+)
+
+func main() {
+	fmt.Fprintln(os.Stderr, "[11933:11933:0618/141553.795857:ERROR:edid_parser.cc(102)] Too short EDID data: manufacturer id")
+	fmt.Fprintln(os.Stderr, "[11933:11933:0618/141553.796025:ERROR:edid_parser.cc(102)] Too short EDID data: manufacturer id")
```

Comment:
Just use a multiline string here?
```
`hi
there`
```

Reply:
I tried looking up why ReadByte(delimiter) didn't work as expected with raw literals, couldn't find anything helpful, left it for now, will look at it later